ui/src/components/loading.tsx

```diff
@@ -38,21 +41,36 @@ export default function Loading(props: Props) {
     return <>{props.children}</>;
   }
 
-  return <Box className="loading">{showStatus(visible, props.status)}</Box>;
+  return (
+    <Box className={props.size === "small" ? "loading-small" : "loading"}>
+      {showStatus(visible, props.status, props.size)}
+    </Box>
+  );
 }
 
-function showStatus(visible: boolean, status?: Status): ReactNode {
+function showStatus(
+  visible: boolean,
+  status?: Status,
+  size?: string
+): ReactNode {
   if (status?.error && !status?.isPending) {
+    const errorMessage = status.error
+      ? `An error has occurred: ${status.error.message}`
+      : "An unknown error has occurred.";
+    if (size === "small") {
+      return (
+        <Tooltip title={errorMessage} arrow={true}>
+          <ErrorIcon />
+        </Tooltip>
+      );
+    }
+
     return (
       <>
         <Typography variant="h2">Error</Typography>
-        {status.error?.message ? (
-          <Typography paragraph>
-            An error has occurred: <b>{status.error.message}</b>
-          </Typography>
-        ) : (
-          <Typography paragraph>An unknown error has occurred.</Typography>
-        )}
+        <Typography paragraph>
+          <b>{errorMessage}</b>
```

Comment:
The bold was formatting the error portion of the message. Just remove it.

Reply:
Done.